webapp/views.py

```diff
@@ -399,15 +399,26 @@ def advantage_view():
     new_subscription_start_date = None
     new_subscription_id = None
     open_subscription = flask.request.args.get("subscription", None)
+    is_staging = flask.request.args.get("ua_staging", False)
+
     stripe_publishable_key = os.getenv(
-        "STRIPE_PUBLISHABLE_KEY", "pk_live_68aXqowUeX574aGsVck8eiIE"
+        "STRIPE_LIVE_PUBLISHABLE_KEY", "pk_live_68aXqowUeX574aGsVck8eiIE"
     )
 
+    api_url = flask.current_app.config["CONTRACTS_LIVE_API_URL"]
+
+    if is_staging:
```

Comment:
If we are serving a page built with staging data, we absolutely have to have to have to include a `x-robots-tag: noindex` header (that's from memory, please check the syntax). These pages *must not* be indexed by Google.

Reply:
Ah Albert suggested putting it in the html. I prefer the http header solution I think, but either will work.

Reply:
I've added it in the head as you suggested.